shell/platform/windows/flutter_windows_engine_unittests.cc

```diff
@@ -1169,5 +1170,30 @@ TEST_F(FlutterWindowsEngineTest, ChannelListenedTo) {
   }
 }
 
+TEST_F(FlutterWindowsEngineTest, ReceivePlatformViewMessage) {
```

Comment:
From my understanding, this file is more for engine unit tests.  Since this launches & runs a full app, should we move this to the integration tests in `flutter_windows_unittests.cc`?

I don't feel strongly about this, feel free to keep as is.

Reply:
It seems to me that this test is quite similar in most regards to the handful of unit tests immediately preceding it in this file. Would you disagree?